### DISCUSSION TITLE
RMG-Cat meetings agenda/notes

### DISCUSSION THREAD

**Discussion by rwest** [ReactionMechanismGenerator/RMG-Py#2101] [General]
This discussion topic is an attempt to organize, in a single thread, agenda items, meeting notes, etc. for heterogeneous catalysis topics in RMG. Generally these are discussed in the #rmg-cat channel in slack, and in biweekly meetings on Monday afternoons.  Subscribe if you're interested, mute if you're not, then update the thread in between meetings with agenda items that you want to discuss.

If it turns out a discussion topic thread is not the best place for this, we'll move elsewhere.

**Comment by Tingchenlee** [comment]
#### Monday, May 10, 2021
I had an issue with ```NO2``` bidentate recently. I'd opened RMG-Py [issue 2122](https://github.com/ReactionMechanismGenerator/RMG-Py/issues/2122
) and RMG-database [issue 477](https://github.com/ReactionMechanismGenerator/RMG-database/issues/477) to report the error messages. One question for our surface families is that we didn’t add the forbidden blocks of radicals and charged species in every family but I'm not sure if it’s necessary to do this.

**Reply by rwest** [reply]
This turned into a bigger discussion of resonance on a surface.
We propose a concerted "hackathon" in late May to figure it out.

To start with, we should all read the paper on resonance in RMG: https://doi.org/10.1002/kin.21307

**Reply by rwest** [reply]
@alongd we may want your guidance on this one. Resonance of adsorbates is a thorny issue in a can of worms that we've been kicking down the road, if you'll pardon me for butchering three metaphors.

**Reply by alongd** [reply]
Couldn't be happier to contribute. I'll join the rmg-cat channel

**Comment by rwest** [comment]
Radicals on the surface:
We should forbid adsorbates with radicals, a global constraint.

**Reply by alongd** [reply]
Could you elaborate? Do you expect radicals to react faster than they are adsorbed? Or will they no longer be radicals once they are adsorbed due to an electron transfer?

**Reply by rwest** [reply]
The latter. Something like: any unpaired electron will pair up with one from the infinite sea of electrons in the metal, and form a bond.

**Comment by Tingchenlee** [comment]
**June 7**
 - RMG-Py PR
      - SurfaceArrhenius <=> StickingCoefficient #2148 (David)
      - Training reactions constraints #2145  (David)
      - added method to write rate const to HTML for sticking rxns #2140 (Chris)
      - Adding in coverage dependent effects to surface kinetics #2090 (Emily)
 - RMG-database PR
      -  Add training reactions and enlarge the nodes in the tree for some surface reaction families #479 (Ting-Chen)

**Comment by rwest** [comment]
Coverage Dependent Kinetics 
Emily's PR https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2090
Should enable:
- coverage-dependent kinetics to be stored in a seed mechanism or reaction library
- used during simulations and to evaluate fluxes
- output into Chemkin file

Very nearly ready, as of 2021-06-07

**Reply by rwest** [reply]
Emily says ready to merge 2021-06-21. Richard was working on it last week and will resume.

**Comment by rwest** [comment]
Coverage Dependent Thermochemistry

@jongyoonbae is adding this to Cantera https://github.com/Cantera/enhancements/issues/59
and is making good progress, as of 2021-06-07. Hopefully will be ready to test in a couple of weeks. Then we can start having RMG create cantera files with this data

**Comment by rwest** [comment]
Adsorption families should have all training kinetics as Sticking Coefficients

PR https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2148

- If data are published as surface Arrhenius, we should translate into sticking coefficients before putting into the training reactions of a family. 
- if a user enters it as SurfaceArrhenius, the database tests should complain, and give a helpful error message that includes the conversion to StickingCoefficient
- when doing database tests, we could also check that they're between 0 and 1. There are "reasonable" sanity checks for the gas phase, we could base it off.  Chris is working on this.
-  things entered in the reverse direction (i.e. desorption) will be a SurfaceArrhenius with units of 1/s. When reversing these to add to the training data at database load time, we will need converting into a StickingCoefficient, not just a reversed SurfaceArrhenius.

**Comment by rwest** [comment]
Database tests could check things are reasonable

eg.
- sticking coefficients between 0 and 1
- rates are reasonable order of magnitude. Print warnings

**Comment by rwest** [comment]
Surface reaction rates could be on a per site basis not per surface area basis.

- Cantera thinks in per-area basis, but much (most??) of the catalysis community uses a per-site basis.
- CatMAP (we think) uses per-site
- should we propose a Cantera enhancement to accept both?
- should RMG think internally on a per-site basis?
    - is it easier to estimate things this way?
    - is more data published this way?

**Comment by rwest** [comment]
How to deal with data from multiple metals and facets??

Since we introduced the ability to store metal and facet as attributes on data, we don't yet do much with it.

Step 1.
When choosing an "exact match" from a training reaction or library, we should restrict it to correct metal and facet. Or allow the user to choose what to do about it. Pick the right one if it exists? 

Step 2.
Allow user to restrict which training reactions are used. David's PR https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2145

Step 3.
Scale the kinetic barriers from the "wrong" metal (but right facet), using LSR and BEP or BlowersMasel.  Not too hard for library reactions. Could also do for training reactions (but not the auto-generated trees).

**Comment by rwest** [comment]
@bjkreitz has a bunch of BEP relations for surface reaction families that he will share. This was collected for the methanation study.
